tests/psbt.rs

```diff
@@ -0,0 +1,459 @@
+
```

Comment:
Needs a module doc here i.e., `//! PSBT test vectors from BIP-174` or something like that.